staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -247,7 +280,7 @@ func (cfgCtlr *configController) updateObservations() {
 func (cfgCtlr *configController) Run(stopCh <-chan struct{}) error {
 	defer cfgCtlr.configQueue.ShutDown()
 	klog.Info("Starting API Priority and Fairness config controller")
-	if ok := cache.WaitForCacheSync(stopCh, cfgCtlr.plInformerSynced, cfgCtlr.fsInformerSynced); !ok {
+	if ok := cfgCtlr.WaitForCacheSync(stopCh); !ok {
```

Comment:
This is usually expressed inline. Let's be consistent with other controllers and put the standard if-block back here.

Reply:
No, the fight test does _not_ invoke `Run`, but it needs the wait-for-cache-sync behavior.